modules/bling/bling.sh

```diff
@@ -13,5 +13,5 @@ find "$PWD" -type f -exec chmod +x {} \;
 
 for ITEM in "${INSTALL[@]}"; do
     echo "Pulling from bling: $ITEM"
-    eval "$PWD/$ITEM.sh"
+    eval "$PWD/$ITEM".sh
```

Comment:
I don't get this error, the `.sh` is clearly there but it's not in the error. I tried both the old and new snippet locally and both worked..

Reply:
From the empty space,, it could be that $ITEM has a newline
```
=== Launching module of type: bling ===
Pulling from bling: fonts

/tmp/modules/bling/bling.sh: line 16: /tmp/modules/bling/installers/fonts: No such file or directory
``

Reply:
That was it. Seems that all variables in all arrays contain a trailing newline, but that doesn't cause big issues anywhere else.